test/suite/AuthSuite.scala

```diff
@@ -53,13 +50,8 @@ trait AuthSuite extends ScalaFutures {
                |  ]
                |}""".stripMargin
 
-  def authenticate(app: Application)(implicit ec: ExecutionContext): Future[HeaderCarrier] = {
-
+  def authenticate(app: Application)(implicit ec: ExecutionContext): Future[Map[String, Seq[String]]] = {
```

Comment:
The signature of this method has changed but I can't see what it is called by in this commit (no changes are present that use this method). What was the reason for it changing?

Reply:
This trait is not used anywhere and it could be even removed. But I was thinking that maybe it's a good reminder of the payload returned by Auth. This is why I did not remove it (but we can still do it if we want).
However `HeaderCarrierConverter.fromHeadersAndSession` is deprecated, so I just removed it.

Reply:
Ok, I'd of probably just removed it if its redundant code but no big deal